packages/nimble-components/src/table/specs/table-column-specs/table-column-elapsed-time-hld.md

```diff
@@ -0,0 +1,177 @@
+# Elapsed Time Table Column
+
+## Overview
+
+The elapsed time table column will provide a way to visualize numeric data that represents a timespan. A 'timespan' is simply a numeric value representing a total number of seconds (i.e. a value of 5 means '5 seconds').
+
+### Background
+
+[Work Item](https://ni.visualstudio.com/DevCentral/_workitems/edit/2526116)
```

Comment:
You should change this link (or add a second link) to the nimble issue rather than the azdo issue: https://github.com/ni/nimble/issues/1569